pkgs/misc/ghostscript/default.nix

```diff
@@ -1,12 +1,13 @@
-{ stdenv, lib, fetchurl, pkgconfig, zlib, expat, openssl, autoconf
+{ config, stdenv, lib, fetchurl, pkgconfig, zlib, expat, openssl, autoconf
 , libjpeg, libpng, libtiff, freetype, fontconfig, libpaper, jbig2dec
 , libiconv, ijs
-, x11Support ? false, xlibsWrapper ? null
-, cupsSupport ? false, cups ? null
+, cupsSupport ? config.ghostscript.cups or (!stdenv.isDarwin), cups ? null
+, x11Support ? cupsSupport, xlibsWrapper ? null # with CUPS, X11 only adds very little
```

Comment:
Somewhat borderline on that… (I do not oppose that, but do not actively suport and will wait longer for objections to this change before merging)

Reply:
But that is exactly what was written in `all-packages.nix` before this change.

Reply:
Hm, I felt this was a common thing to do, `grep -rE '.*Support \? .*Support'` gives a lot of results.

Reply:
I tried to comment on the entire two-line combo — `…Support ? …Support` but together with `…Support ? config.…`

Reply:
Ah, I see. Well, a) it is just another variable binding, so not that much of a difference, IMHO b) I don't see any other options in this case as applying `config.ghostscript or {}` in `all-packages.nix` will do the wrong thing.